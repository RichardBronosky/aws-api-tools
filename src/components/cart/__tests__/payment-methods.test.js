import React from 'react'
import renderer from 'react-test-renderer'
import PaymentMethods from '../cart-parts/payment-methods'

describe('PaymentMethods', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<PaymentMethods />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
