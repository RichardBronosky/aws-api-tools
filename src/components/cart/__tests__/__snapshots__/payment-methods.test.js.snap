// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaymentMethods renders correctly 1`] = `
Array [
  <h4
    className="bold"
  >
    We Accept
  </h4>,
  <img
    alt="Accepted payment types: Discover, Mastercard, Visa, Amex, Rooms to go Credit, PayPal, Affirm"
    className="paymentLogos"
    src="test-file-stub"
  />,
]
`;
